flasc/model_fitting/floris_tuning.py

```diff
@@ -62,11 +49,28 @@ def sweep_velocity_model_parameter_for_overall_wake_losses(
     ws_max=50.0,
     df_freq=None,  # Not yet certain we will use this
 ):
-    """
-    Sweep the parameter in FLORIS using the values in value_candidates, and compare to
-    SCADA data in df_scada_in using the overall_wake_loss
-    """
+    """Sweep the parameter in FLORIS using the values in value_candidates.
+
+    Compare to SCADA data in df_scada_in using the overall_wake_loss
 
+    Args:
+        parameter (str): The parameter to sweep
+        value_candidates (list): The values to sweep
+        df_scada_in (DataFrame): The SCADA data
+        fm_in (floris.tools.Floris): The FLORIS model
+        ref_turbines (list): The reference turbines
+        test_turbines (list): The test turbines
+        param_idx (int): The parameter index
+        yaw_angles (np.ndarray): The yaw angles
+        wd_min (float): The minimum wind direction
+        wd_max (float): The maximum wind direction
+        ws_min (float): The minimum wind speed
+        ws_max (float): The maximum wind speed
+        df_freq (DataFrame): The frequency data
+
+    Returns:
+        (np.ndarray, np.ndarray): The FLORIS wake losses and the SCADA wake loss
```

Comment:
Not sure whether this should also be a list:
```
 Returns:
        (np.ndarray): FLORIS wake losses
        (np.ndarray): SCADA wake loss(es?)

Reply:
Fixing to the style I've seen in the guide of declaring the return a Tuple